src/raw/mod.rs

```diff
@@ -1788,7 +1788,7 @@ impl RawTableInner {
     ///
     /// This function does not check if the given element exists in the table. Also,
     /// this function does not check if there is enough space in the table to insert
-    /// a new element. Caller of the funtion must make ensure that the table has at
+    /// a new element. The caller of the function must make ensure that the table has at
```

Comment:
This should either be "make sure" or "ensure".
```suggestion
    /// a new element. The caller of the function must make sure that the table has at
```